src/DependencyInjection/SymfonyCastsVerifyEmailExtension.php

```diff
@@ -31,6 +31,9 @@ public function load(array $configs, ContainerBuilder $container): void
 
         $helperDefinition = $container->getDefinition('symfonycasts.verify_email.helper');
         $helperDefinition->replaceArgument(4, $config['lifetime']);
+        $helperDefinition->replaceArgument(5, $config['use_email']);
+        // expose configuration value so it can be used in controllers
+        $container->setParameter('symfonycasts.verify_email.use_email', $config['use_email']);
```

Comment:
Can you explain why you think this needs to be accessible?

Reply:
My reason for this option to be exposed is to give the user the opportunity to use the "old" way how user verification worked or the alternate way by including the email.